src/app/admin-manage/admin-manage.controller.js

```diff
@@ -152,13 +152,14 @@
             });
         }
 
-        function open(consentId, url, electionId) {
+        function open(consentId, url, electionId, showConsent) {
```

Comment:
One way to make usages of this method cleaner is to set the default argument. For example, if this were
```
function open(consentId, url, electionId, showConsent = false)
```
Then lines 51-54 would not need to change in `admin-manage-dul.html`. I would argue that those lines could be further cleaned up if the default for `electionId` was also null, and you would not need to pass in nulls in those same lines.

Reply:
It seems that default functions parameters are only available in ES6. If we want to implement this we need to update this library.

Reply:
Oh well - sorry about that. The alternative approaches to default arguments are not really worth the effort here, IMHO.